tests/integration/targets/android_sdk/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# java >= 17 is not available in RHEL and CentOS7 repos, which is required for sdkmanager to run
+- name: Bail out if not supported
+  when:
+  - "ansible_os_family == 'RedHat' and ansible_distribution_version is version('8.0', '<')"
```

Comment:
And since you liked and adopted it so fast ;-) a quick question here: aren't there any other minimum requirements for other platforms?

Reply:
Not sure that I understood you. All other OS variants pass on CI, meaning that java 17 is available there, so AFAIK only RHEL 7 and CentOS 7 are subjects to be left out.

Reply:
another brain fart, never mind.